src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -16,9 +24,18 @@
 {% endif %}
 
 {% if archivedVersions %}
+    {{ form_start(archivedActionsForm, {'attr': {'class': 'ez-js-standard-form'}}) }}
+
     <div class="ez-list-toolbar">
         <h2 class="ez-list-toolbar-label">{{ 'locationview.versions.archived.versions'|trans()|desc('Archived versions') }}</h2>
+        <div class="ez-list-toolbar-tools">
+            {{ form_row(archivedActionsForm.new_draft, {'label': 'locationview.versions.new.draft.selected.version'|trans()|desc('New draft based on selected version'), 'attr': {'class': 'ez-button ez-button-secondary'}}) }}
+            {{ form_row(archivedActionsForm.delete, {'label': 'locationview.versions.remove.selected.archived.version'|trans()|desc('Remove selected archived version(s)'), 'attr': {'class': 'ez-button ez-button-negative'}}) }}
```

Comment:
It should be `desc('Delete')` instead of `desc('Remove selected archived version(s)')` (_cf._ mockup)